charts/rbs/README.md

```diff
@@ -29,6 +29,7 @@ Chart for Resource Buffering Service
 | imageConfig.imagePullPolicy | string | `"Always"` |  |
 | imageConfig.repository | string | `"gcr.io/terra-kernel-k8s/terra-rbs"` | Image repository |
 | imageConfig.tag | string | The chart's appVersion value will be used | Image tag. |
+| pool.configPath | string | `nil` | The configuration files pools and resources. Currently all pool configs are inside Java binary under resources/ so the value is a relative path from resource/, for example: config/dev |
```

Comment:
optional: maybe even prefix this with `rbs` so `rbs.pool.configPath`. Everything else is at a whoe other level of specification. looking at the other PR, I was very unsure what a pool.configPath related to. The comment here is good.

Reply:
Done